about screen layout orientation changes

V9vek opened the pull request:
The guy who added the About Screen didn't add the `NestedScrollView` for the layout changes when we change to landscape orientation

`Previous About Screen:`
![prev_about](https://user-images.githubusercontent.com/40730402/136022159-23a1399f-23f4-4db0-8714-06790c264f9e.png)


`After About Screen I have made changes:`

https://user-images.githubusercontent.com/40730402/136022005-95bf407a-ba46-47dc-bb93-2615dca73ae1.mp4

V9vek commented:
I don't know why the video is in pink 😆 
But please review and check on your device and do let me know

rishuyadav commented:
Thats gr8 @V9vek 👍🏻can u please create an issue also regarding this, then we can merge ur pr with reference to that issue, plus u get points for issues also🚀

gaganmalvi commented:
Video works fine here, good job. Please create an issue and reference it here, we'll merge it.

V9vek commented:
I have raised a issue regarding this, please check

rishuyadav commented on app/src/main/res/layout/activity_about.xml:
```diff
@@ -74,45 +50,75 @@
+                android:layout_marginEnd="16dp"
+                android:fontFamily="@font/poppins"
+                android:gravity="center"
+                android:text="ACM-VIT, one of the most reputed and distinguished technical chapter in VIT-Vellore has been working with vision and determination since it’s inception in 2009."
```
This string can be extracted from string res

V9vek replied:
okay I am doing that and will push changes
anything else?

rishuyadav replied:
Nope nothing else, just extract all hardcoded strings

rishuyadav commented:
Hey @V9vek pls save strings and extract from there, then the pr is good to merge 👍

V9vek commented:
Removed hardcoded strings and extracted from string res file 
Please check it out and merge it with the required `hacktoberfest` tags

Thanks 🙏🏼

rishuyadav commented:
Thanks a lot @V9vek for contributing 
Happy Hacktoberfest 🥳
